app/services/mixpanel_service.rb

```diff
@@ -309,12 +309,10 @@ def data_from_gyr_intake(intake)
         had_earned_income: intake.had_earned_income? ? "yes" : "no",
         state: intake.state_of_residence,
         zip_code: intake.zip_code,
-        needs_help_2023: intake.needs_help_2023,
-        needs_help_2022: intake.needs_help_2022,
-        needs_help_2021: intake.needs_help_2021,
-        needs_help_2020: intake.needs_help_2020,
-        needs_help_2019: intake.needs_help_2019,
-        needs_help_2018: intake.needs_help_2018,
+        needs_help_current_year: intake.needs_help_current_year,
+        needs_help_previous_year_1: intake.needs_help_previous_year_1,
+        needs_help_previous_year_2: intake.needs_help_previous_year_2,
+        needs_help_previous_year_3: intake.needs_help_previous_year_3,
         needs_help_backtaxes: intake.needs_help_with_backtaxes? ? "yes" : "no",
```

Comment:
just curious -- was max (or anyone) using this mixpanel `needs_help` data?

Reply:
I asked Max if it was okay to make this change, I didn't ask specifically if he was using the needs_help data but might be a good idea to mention it to him that this data wasn't here before and the other needs_help columns were basically trash. I doubt anyone would have been using it without it raising red flag as it would have just returned unfilled for every intake